projects/angular-components/src/lib/search/search.component.html

```diff
@@ -9,7 +9,7 @@
     (input)="_onInteractionEvent($event)"
     #input
   />
-  @if (!disabled) {
+  @if (!noFocus && !disabled) {
```

Comment:
This is a bit of the same. I think it'd be nicer if you can see, this bit of code runs when `(focussed && enabled)`.

Reply:
Where would you get the `focussed` and `enabled` variables from?
These are also input variables based on html:input elements which also have the disabled property.
For noFocus, currently it works nice to just do `<ff-search noFocus></ff-search>` instead of having to add `<... focus>` while wanting to use what is now default

Reply:
I changed `noFocus` to `keyFocusEnabled`